crates/axon-jobs/src/migrations/0018_unified_jobs_observability.sql

```
@@ -0,0 +1,105 @@
+    stage_id TEXT,
+    phase TEXT NOT NULL,
+    status TEXT NOT NULL,
+    severity TEXT NOT NULL,
+    visibility TEXT NOT NULL,
+    message TEXT NOT NULL,
+    timestamp TEXT NOT NULL,
+    details_json TEXT NOT NULL DEFAULT '{}',
+    UNIQUE(job_id, sequence)
+);
+
+CREATE INDEX job_events_job_sequence_idx ON job_events(job_id, sequence);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)